internal/executor/metrics/metrics_unsupported.go

```diff
@@ -8,11 +9,15 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func Run(ctx context.Context, logger logrus.FieldLogger) (chan *api.ResourceUtilization, chan error) {
-	resultChan := make(chan *api.ResourceUtilization, 1)
-	errChan := make(chan error, 1)
+type Result struct {
+	Errors              []error
```

Comment:
I wonder if it make sense to deduplicate the errors before sending them. There might be dozens of identical errors. @edigaryev what do you think?

Reply:
Good idea, added error deduplication in https://github.com/cirruslabs/cirrus-ci-agent/pull/200/commits/48d8698d6da55828aa565cfa77a97ac2e83529fd.